src/ui/storybook/StoryContainer.tsx

```diff
@@ -11,6 +11,8 @@ type Props = WithBackgroundProps & {
   title?: string
 }
 
+const CaptionInformation = 'Background is not a part of this story'
```

Comment:
le background fait parti de la story mais pas du composant

```suggestion
const CaptionInformation = 'Background is not a part of this component'
```